tasks-repository.go

```diff
@@ -1,65 +1,55 @@
 package main
 
 import (
-	"database/sql"
 	"fmt"
+	"strconv"
 )
 
-// Returns all tasks in SQL database
-func getTasks_sql() ([]Task, error) {
+// Returns all tasks in SQL database that match input filter
+func getTasksByFilter_sql(searchedID string, searchedTitle string, searchedComplete string) ([]Task, error) {
 	var taskList []Task
 
-	tasks, err := db.Query("SELECT * FROM tasks")
-	if err != nil {
-		return taskList, err
-	}
-
-	for tasks.Next() {
-		var tsk Task
+	query := "SELECT * FROM tasks WHERE 1=1"
```

Comment:
Why add the condition WHERE 1=1? This feels redundant and if there isn't a reason for it, lets remove it